lib/server/routes/health.js

```diff
@@ -23,7 +27,7 @@ HealthRouter.prototype.health = function(req, res) {
 
 HealthRouter.prototype._definitions = function() {
   return [
-    ['GET', '/health', this.health]
+    ['GET', '/health', this.getLimiter(limiter(1000)), this.health]
```

Comment:
Currently this endpoint is memory only without any network calls, calling to redis to check the limiter would be the most expensive thing. We only use this internally so we could add a token that's necessary to use it, that's typically common for these types of endpoints.